config/settings.py

```diff
@@ -39,3 +39,6 @@ def abspath(relative_file_path):
 
 IPFS_DOMAIN = os.environ.get('IPFS_DOMAIN')
 IPFS_PORT = os.environ.get('IPFS_PORT')
+
+REDIS_URL = os.environ.get('REDIS_URL')
+CELERY_DEBUG = os.environ.get('CELERY_DEBUG', False)
```

Comment:
1. I don't think this will work. Environment variables read from .env files will be strings, not booleans. So would need to do `os.environ.get('CELERY_DEBUG') == 'True'` or something like that. Or am I mistaken?

2. Can we add `CELERY_DEBUG` to `dev.env`? Can just have it set to false: `CELERY_DEBUG=False`

Reply:
Gotcha! fixing and updating everything. Although celery type coerces this and it will work, this is bad practise and will fix this.

Reply:
Ah ok, didn't know that. 👍